Add readme file and a master makefile

VeaNika opened the pull request:
The master makefile allows to run `make` in the root of GBARunner3 workspace folder, and will pack the following inside the `out` folder:

```
out\_gba\configs
out\_gba\GBARunner.json
out\_GBARunner3.nds
```

A default config file was added into the repo named as `_GBARunner3.json`, which gets moved and renamed properly as `out\_gba\GBARunner.json`, just after the makefile copies the `config` folder to `out`.

This will also close https://github.com/Gericom/GBARunner3/issues/153

The readme file still needs review, feel free to ping me on Discord to ask me change anything to your likings!

VeaNika committed:
Added a default GBARunner3.json configuration file

VeaNika committed:
Added a master Makefile

VeaNika committed:
Added GBARunner3 Hi-res logo (Thanks to @Dartz150)

VeaNika committed:
Added initial Readme

VeaNika committed:
Added and corrected some additional info to the readme file.

lifehackerhansol commented:
You may want to change GitHub Actions workflow to adapt to this new Makefile as well. Currently it's doing what this Makefile does, but separately.

Probably best if you test the changes separately from the PR, then clean up the commits and cherry-pick it here.

It's not "needed" since the workflow should still work otherwise, but it'd be nice to have less logic in there.

VeaNika committed:
Updated GitHub Actions workflows

VeaNika commented:
@lifehackerhansol I totally forgot about the GitHub Actions workflow, thanks for the reminder! It's fixed now and it should work as expected, since the old workflow wasn't moving the new `GBARunner3.json` default settings file to `out/_gba`

VeaNika committed:
More QoL fixes and additional information.

Deletecat committed:
readme: change gbarunner3 frontend version

The twilight menu version of the frontend
expects GBAR3 to be in '/_nds/TWiLightMenu/emulators'.
However, the setup instructions tell the user to place
GBAR3 on the root of their SD card, which means the frontend
will not be able to find it. Swapping out the twilight
frontend with the normal frontend will fix this issue.

VeaNika committed:
Merge pull request #6 from Deletecat/feature/add-readme-and-master-makefile

readme: change gbarunner3 frontend version

VeaNika committed:
Merge branch 'Gericom:develop' into feature/add-readme-and-master-makefile

VeaNika committed:
Updated readme and config with the new options and their documentation
- Improved overall readability of the readme.
